content/en/docs/reference/kube-scheduler/profiles.md

```diff
@@ -0,0 +1,174 @@
+   
+## Scheduling plugins
+
+The following plugins, enabled by default, implement one or more of these
+extension points:
+
+
+- `DefaultTopologySpread`: Favors spreading across nodes for Pods that belong to
+  services, replica sets and stateful sets.
+  Extension points: *PreScore*, *Score*.
+- `ImageLocality`: Favors nodes that already have the container images that the
+  Pod runs.
+  Extension points: *Score*.
+- `TaintToleration`: Implements
+  [taints and tolerations](/docs/concepts/configuration/taint-and-toleration/).
+  Implements extension points: *Filter*, *Prescore*, *Score*.
+- `NodeName`: Checks if a Pod spec node name matches the current node.
+  Extension points: *Filter*.
+- `NodePorts`: Checks if a node has free ports for the requested Pod ports.
+  Extension points: *PreFilter*, *Filter*.
+- `NodePreferAvoidPods`: Scores nodes according to the node annotation
+  `scheduler.alpha.kubernetes.io/preferAvoidPods`.
+  Extension points: *Score*.
+- `NodeAffinity`: Implements
+  [node selectors](/docs/concepts/configuration/assign-pod-node/#nodeselector)
+  and [node affinity](/docs/concepts/configuration/assign-pod-node/#node-affinity).
+  Extension points: *Filter*, *Score*.
+- `PodTopologySpread`: Implements
+  [Pod topology spread](/docs/concepts/workloads/pods/pod-topology-spread-constraints/).
+  Extension points: *PreFilter*, *Filter*, *PreScore*, *Score*.
+- `NodeUnschedulable`: Filters out nodes that have `.spec.unschedulable` set to
+  true.
+  Extension points: *Filter*.
+- `NodeResourcesFit`: Checks if the node has all the resources that the Pod is
+  requesting.
+  Extension points: *PreFilter*, *Filter*.
+- `NodeResourcesBallancedAllocation`: Favors nodes that would obtain a more
+  balanced resource usage if the Pod is scheduled there.
+  Extension points: *Score*.
+- `NodeResourcesLeastAllocated`: Favors nodes that have a low allocation of
+  resources.
+  Extension points: *Score*.
+- `VolumeBinding`: Checks if the node has or if it can bind the requested
+  volumes.
+  Extension points: *Filter*.
+- `VolumeRestrictions`: Checks that volumes mounted in the node satisfy
+  restrictions which are specific to the volume provider.
+  Extension points: *Filter*.
+- `VolumeZone`: Checks that volumes requested satisfy any zone requirements they
+  might have.
+  Extension points: *Filter*.
+- `NodeVolumeLimits`: Checks that CSI volume limits can be satisfied for the
+  node.
+  Extension points: *Filter*.
+- `EBSLimits`: Checks that EBS volume limits can be satisfied for the node.
+  Extension points: *Filter*.
+- `GCEPDLimits`: Checks that GCP-PD volume limits can be satisfied for the node.
+  Extension points: *Filter*.
+- `AzureDiskLimits`: Checks that Azure disk volume limits can be satisfied for
+  the node.
+  Extension points: *Filter*.
+- `InterPodAffinity`: Implements
+  [inter Pod affininity and antiaffinity](/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
```

Comment:
Combined typo fix and reword:
```suggestion
  inter-Pod [affinity and anti-affinity](/docs/concepts/configuration/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
```

Reply:
Done, except that I kept `inter-Pod` inside the brackets, because there is a separate section for the more general "affinity and anti-affinity"